apps/api/src/config/index.ts

```diff
@@ -109,6 +109,7 @@ export const config = {
       process.env.DATABASE_READ_REPLICA_URL ||
       process.env.DATABASE_URL ||
       "postgresql://postgres:postgres@localhost:5432/trading_simulator",
+    skipMigrations: process.env.DB_SKIP_MIGRATIONS === "true",
```

Comment:
note: i added this because during local dev, i sometimes have to skip db migrations. e.g., if i have a branch that requires a rebase + nuke my feature branch's drizzle migration files, then i effectively have to recreate that db state, again...unless i can just skip the migrations.

Reply:
There is a feature called `push` in drizzle that can be used in dev time to just apply the schema to the db, ignoring migrations. We should build some tooling to use it and move away from running the migrations on startup since we won't be deploying to hetzner machines long term.

Reply:
Ah looking at this more, seems like my comment is unrelated. Ignore.